src/pages/komodo-defi-framework/api/common_structures/activation/index.mdx

```diff
@@ -144,16 +144,68 @@ Contains information electrum servers for coins being used in `Electrum` or `Lig
 
 The `EvmNode` object includes the following items for a given coin or token:
 
-| Parameter | Type    | Description                                                                                                         |
-| --------- | ------- | ------------------------------------------------------------------------------------------------------------------- |
-| url       | string  | URL of an RPC node                                                                                                  |
-| gui\_auth | boolean | Optional, defaults to `false`. Must be set to `true` to access RPC nodes run officially by the Komodo Platform team |
+| Parameter    | Type    | Description                                                                                                         |
+| ------------ | ------- | ------------------------------------------------------------------------------------------------------------------- |
+| url          | string  | URL of an RPC node                                                                                                  |
+| komodo\_auth | boolean | Optional, defaults to `false`. Must be set to `true` to access RPC nodes run officially by the Komodo Platform team |
+
+#### v2.1.0 (current release)
 
 <CollapsibleSection expandedText="Hide Example" collapsedText="Show Example">
   ```json
   {
       "url": "http://eth1.cipig.net:8555",
-      "gui_auth": false
+      "komodo_proxy": false
+  }
+  ```
+</CollapsibleSection>
+
+<Note>
+  Prior to the v2.2.0 release, the `gui_auth` field was used to access RPC nodes run officially by the Komodo Platform team. This field was replaced with the `komodo_proxy` field in the v2.2.0 release.
+</Note>
+
+#### v2.2.0+
+
+<CollapsibleSection expandedText="Hide Example" collapsedText="Show Example">
+  ```json
+  {
+      "url": "https://nucleus-rpc.alpha.komodo.earth",
+      "komodo_proxy": true
+  }
+  ```
+</CollapsibleSection>
+
+### TendermintNode
+
+The `TendermintNode` object includes the following items for a given coin or token:
+
+| Parameter    | Type    | Description                                                                                                         |
+| ------------ | ------- | ------------------------------------------------------------------------------------------------------------------- |
+| url          | string  | URL of an RPC node                                                                                                  |
+| komodo\_auth | boolean | Optional, defaults to `false`. Must be set to `true` to access RPC nodes run officially by the Komodo Platform team |
```

Comment:
I would update this with something like "Must be set to `true` to access RPC nodes run behind [komodo-defi-proxy](https://github.com/KomodoPlatform/komodo-defi-proxy)"

Reply:
done, thanks.